datastruct.go

```diff
@@ -1,7 +1,11 @@
 package main
 
 type UpdateResourceRequest struct {
-	IpAddress               string `json:"ipAddress"`
-	CertificateProviderType string `json:"certificateProviderType"`
-	CertificateExpiryDate   string `json:"certificateExpiryDate"`
+	IpAddress                string `json:"ipAddress"`
+	CertificateProviderType  string `json:"certificateProviderType"`
+	CertificateExpiryDate    string `json:"certificateExpiryDate"`
+	HwLastRebootReason       string `json:"hw-last-reboot-reason"`
+	WebpaInterfaceUsed       string `json:"webpa-interface-used"`
+	WebpaLastReconnectReason string `json:"webpa-last-reconnect-reason"`
```

Comment:
When we are sending request to update resource, we should follow camel casing for request body variables.
Please make the necessary changes and sync with Nikhil for the same.